### PR TITLE
Updated Postgres Config

### PR DESCRIPTION
**Pull request by D-Bits**
Updated docker-compose file to use volumes to persist Postgres data. Also updated environment variables to use required naming conventions.

### CODE REVIEW

**Comment by asgray** [comment]
was the React container removed from the compose file intentionally?

**Comment by denisemauldin** [comment]
Looks like this needs to be rebased off the devroastproject:develop branch because the docker-compose.yml doesn't have the react container as @asgray identified.

The ticket/issue didn't include anything about removing the example.env file, so why was that done?

The README.md file needs to be updated to specify the new parameters that start with `POSTGRES` instead of the current values that start with `DB`.

File: docker-compose.yml [hunk 1]
```diff
@@ -20,8 +20,10 @@ services:
 
   postgres:
     image: postgres:12.2-alpine
+    networks:
+      [webservice_network]
     volumes:
-      - ./pgdata/:/var/lib/postgresql/data/devroast
+      - devroast:/var/lib/postgresql/data/
```
Comment (denisemauldin) [inline docker-compose.yml:44]: Why is this updating to have a named volume with no definition instead of a directory?

File: devroast/settings.py [hunk 2]
```diff
@@ -84,11 +84,11 @@
 DATABASES = {
     'default': {
         'ENGINE': 'django.db.backends.postgresql_psycopg2', 
-        'NAME': os.getenv("DB_NAME"),
-        'USER': os.getenv("DB_USER"),
-        'PASSWORD': os.getenv("DB_PASS"),
-        'HOST': os.getenv("DB_HOST"),
-        'PORT': os.getenv("DB_PORT"),
+        'NAME': os.getenv("POSTGRES_NAME"),
```
Comment (denisemauldin) [inline devroast/settings.py:87]: These changes should be documented.

File: env.example [hunk 3]
```diff
@@ -1,4 +0,0 @@
-DEBUG=1
```
Comment (denisemauldin) [inline env.example:1]: Why was this file deleted?

**Review by denisemauldin — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (D-Bits) [reply docker-compose.yml:44]: I found that using a Docker volume instead of hard directory solved earlier problems with bootstrapping the container.
